releases.go

```diff
@@ -49,11 +49,14 @@ func (r *Release) BeforeCreate() error {
 // ReleasesQuery is a Scope implementation for common things to filter releases
 // by.
 type ReleasesQuery struct {
-	// If Provided, an app to filter by.
+	// If provided, an app to filter by.
 	App *App
 
 	// If provided, a version to filter by.
 	Version *int
+
+	// If provided, paginate the results.
+	Limit *int
```

Comment:
If we specify a Range type as a struct, might be nice to just pass that directly here:

```go
// If provided, uses the limit and sorting parameters specified in the range.
Range *headerutil.Range
```